<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scoreFinish</class>
 <widget class="QWidget" name="scoreFinish">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>558</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>691</width>
    <height>558</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>691</width>
    <height>558</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(235, 244, 234);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>281</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QTableWidget" name="scoTable">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>231</width>
     <height>146</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,255,255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
